recipes/libsigcpp/2.10.x/test_package/conanfile.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
```suggestion
```

Reply:
Why should I remove this? https://github.com/conan-io/conan-center-index/blob/b8683bc6a9edb20bfcec2182388245c2abecc7d6/recipes/libjpeg/all/conanfile.py#L1 
Here it is present